spec/views/spree/admin/shared/_order_links.html.haml_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "spec_helper"
+
+describe "spree/admin/shared/_order_links.html.haml" do
+  helper Spree::BaseHelper # required to make pretty_time work
+
+  around do |example|
+    original_config = Spree::Config[:enable_invoices?]
+    example.run
+    Spree::Config[:enable_invoices?] = original_config
+  end
+
+  before do
+    order = create(:completed_order_with_fees)
```

Comment:
do we really need the fees? I bet the simplest of the orders we can get from factories is enough to check we render the actions dropdown. The more objects we create the slower the test is, and we write quite a lot of them.

Reply:
I agree, done!